### PR TITLE
Expose CSV_EXPORT as gui plugin again

### LINKED ISSUE #5049: Lifetime of CopyableLabel can be too short under certain conditions
```
tests/unit_tests/gui/model/test_job_list.py::test_no_cross_talk
  /opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/_pytest/threadexception.py:73: PytestUnhandledThreadExceptionWarning: Exception in thread Thread-154
  
  Traceback (most recent call last):
    File "/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/threading.py", line 932, in _bootstrap_inner
      self.run()
    File "/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/threading.py", line 1254, in run
      self.function(*self.args, **self.kwargs)
    File "/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/ert/gui/ertwidgets/copyablelabel.py", line 72, in restore_text
      self.copy_button.setIcon(QIcon(icon_path))
  RuntimeError: wrapped C/C++ object of type QPushButton has been deleted
  
    warnings.warn(pytest.PytestUnhandledThreadExceptionWarning(msg))
```

Likely it should be created with parent setl to the main window

### PR DESCRIPTION
**Pull request by eivindjahren**
In 8ef5e3148904eaa8ec89e8481334551ed28d704e CSV_EXPORT was removed as it hadn't functioned correctly for a long time, and was believed not used. By user request. It is now back.

This commit fixes the broken issue with get_context_value and adds a test for the plugin tool.

Fixes #5049 

## Pre review checklist

- [x] Added appropriate release note label
- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [x] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).
- [x] Updated documentation
- [x] Ensured new behaviour is tested

Adding labels helps the maintainers when writing release notes. This is the [list of release note labels](https://github.com/equinor/ert/labels?q=release-notes).

### CODE REVIEW

**Comment by eivindjahren** [comment]
jenkins retest this please

File: src/ert/gui/simulation/ensemble_smoother_panel.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ def __init__(self, ert: EnKFMain, notifier: ErtNotifier):
         self._case_selector = CaseSelector(facade, notifier)
         layout.addRow("Current case:", self._case_selector)
 
-        runpath_label = CopyableLabel(text=facade.run_path)
+        runpath_label = CopyableLabel(text=facade.run_path, parent=self)
```
Comment (andreas-el) [inline src/ert/gui/simulation/ensemble_smoother_panel.py:40]: Did you miss one?

```
./src/ert/gui/simulation/ensemble_experiment_panel.py:39:        runpath_label = CopyableLabel(text=self.facade.run_path)
```

**Review by andreas-el — CHANGES_REQUESTED** [review]

File: src/ert/gui/simulation/ensemble_smoother_panel.py (hunk 1 above)
Comment (eivindjahren) [reply src/ert/gui/simulation/ensemble_smoother_panel.py:40]: I clearly did :)

Comment (eivindjahren) [reply src/ert/gui/simulation/ensemble_smoother_panel.py:40]: Fixed

**Pull request merged**